core/src/main/java/com/novoda/merlin/registerable/Registrar.java

```diff
@@ -0,0 +1,63 @@
+package com.novoda.merlin.registerable;
+
+import com.novoda.merlin.Merlin;
+import com.novoda.merlin.MerlinException;
+import com.novoda.merlin.registerable.bind.Bindable;
+import com.novoda.merlin.registerable.connection.Connectable;
+import com.novoda.merlin.registerable.disconnection.Disconnectable;
+
+public class Registrar {
```

Comment:
Is this just like a rename , or there are actually changes in compare to `Registerer`?